plugins/rbac-backend/package.json

```diff
@@ -23,33 +23,33 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.19.8",
-    "@backstage/backend-plugin-api": "^0.6.6",
-    "@backstage/backend-test-utils": "^0.2.7",
-    "@backstage/catalog-client": "^1.4.5",
+    "@backstage/backend-common": "^0.20.1",
+    "@backstage/backend-plugin-api": "^0.6.9",
+    "@backstage/backend-test-utils": "^0.2.10",
+    "@backstage/catalog-client": "^1.5.2",
     "@backstage/catalog-model": "^1.4.3",
     "@backstage/config": "^1.1.1",
-    "@backstage/core-plugin-api": "^1.7.0",
+    "@backstage/core-plugin-api": "^1.8.2",
     "@backstage/errors": "^1.2.3",
-    "@backstage/plugin-auth-node": "^0.4.0",
-    "@backstage/plugin-permission-backend": "^0.5.29",
-    "@backstage/plugin-permission-common": "^0.7.9",
-    "@backstage/plugin-permission-node": "^0.7.17",
+    "@backstage/plugin-auth-node": "^0.4.3",
+    "@backstage/plugin-permission-backend": "^0.5.32",
+    "@backstage/plugin-permission-common": "^0.7.12",
+    "@backstage/plugin-permission-node": "^0.7.20",
     "@dagrejs/graphlib": "^2.1.13",
     "@janus-idp/backstage-plugin-rbac-common": "1.2.1",
     "@janus-idp/backstage-plugin-rbac-node": "1.0.0",
     "casbin": "^5.27.1",
     "express": "^4.18.2",
     "express-promise-router": "^4.1.1",
-    "knex": "^2.0.0",
+    "knex": "^3.0.0",
```

Comment:
@PatAKnight @AndrienkoAleksandr I have updated knex to 3.0 as @backstage/backend-common has bumped knex to 3.0 as well https://github.com/backstage/backstage/blob/master/packages/backend-common/CHANGELOG.md#patch-changes-13 but it looks like knex-mock-client is still running on knex 2.0 and because of that all the tests using this dependency to create a mock-client are giving type issues like below:
```
Type 'typeof MockClient' is not assignable to type 'string | typeof Client | undefined'.
  Type 'typeof MockClient' is not assignable to type 'typeof Client'.
    Types of construct signatures are incompatible.
      Type 'new (config: Config<any>) => MockClient' is not assignable to type 'new (config: Config<any>) => Client'.
        Types of parameters 'config' and 'config' are incompatible.
          Type 'import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex/types/index").Knex.Config<any>' is not assignable to type 'import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex-mock-client/node_modules/knex/types/index").Knex.Config<any>'.
            Types of property 'client' are incompatible.
              Type 'string | typeof import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex/types/index").Knex.Client | undefined' is not assignable to type 'string | typeof import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex-mock-client/node_modules/knex/types/index").Knex.Client | undefined'.
                Type 'typeof Client' is not assignable to type 'string | typeof Client | undefined'.
                  Type 'typeof import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex/types/index").Knex.Client' is not assignable to type 'typeof import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex-mock-client/node_modules/knex/types/index").Knex.Client'.
                    Types of construct signatures are incompatible.
                      Type 'new (config: Config<any>) => Client' is not assignable to type 'new (config: Config<any>) => Client'.
                        Types of parameters 'config' and 'config' are incompatible.
                          Type 'import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex-mock-client/node_modules/knex/types/index").Knex.Config<any>' is not assignable to type 'import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex/types/index").Knex.Config<any>'.
                            Types of property 'client' are incompatible.
                              Type 'string | typeof import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex-mock-client/node_modules/knex/types/index").Knex.Client | undefined' is not assignable to type 'string | typeof import("/Users/divi/janus-idp-backstage-plugins/node_modules/knex/types/index").Knex.Client | undefined'.
                                Type 'typeof Client' is not assignable to type 'string | typeof Client | undefined'.
                                  Types of construct signatures are incompatible.
                                    Type 'new (config: Config<any>) => Client' is not assignable to type 'new (config: Config<any>) => Client'.
                                      Construct signature return types 'Client' and 'Client' are incompatible.
                                        The types returned by 'queryBuilder()' are incompatible between these types.
                                          Type 'QueryBuilder<any, any>' is missing the following properties from type 'QueryBuilder<any, any>': orHavingNull, orHavingNotNull
```
LMK if I should not update rbac-backend plugin as of now or if you have anything else in mind on how this can be handled. Thanks!

Reply:
@AndrienkoAleksandr never mind, I was able to fix the issue. Thanks for looking into it anyways.